EarlGrey.xcodeproj/project.pbxproj

```diff
@@ -1018,6 +1038,8 @@
 				OTHER_CFLAGS = (
 					"-fobjc-arc",
 					"-fobjc-arc-exceptions",
+					"-DNS_BLOCK_ASSERTIONS=1",
```

Comment:
is -DXXX necessary?

You can also build via:
`xcodebuild test OTHER_CFLAGS=XXX
`
Although I am not sure if it appends or replaces whatever is read from the project. You want the former.

Ref: http://clang.llvm.org/docs/CommandGuide/clang.html